core/src/main/scala/kafka/controller/KafkaController.scala

```diff
@@ -900,7 +900,7 @@ class KafkaController(val config: KafkaConfig, zkClient: KafkaZkClient, time: Ti
    * `ControllerContext.partitionsBeingReassigned` must be populated with all partitions being reassigned before this
    * method is invoked to avoid premature deletion of the `reassign_partitions` znode.
    */
-  private def removePartitionsFromReassignedPartitions(partitionsToBeRemoved: Set[TopicPartition]) {
+  def removePartitionsFromReassignedPartitions(partitionsToBeRemoved: Set[TopicPartition]) {
```

Comment:
Make it package private instead of public using ```private[controller] removePartitionsFromReassignedPartitions(partitionsToBeRemoved: Set[TopicPartition])```?

Reply:
fixed.